build/mbuffer/patches/series

```diff
@@ -0,0 +1 @@
+no-gcrypt.patch
```

Comment:
in order to activate the new configure you have to call `autoconf`

Reply:
fixed.